Opcodes/sfont.c

```diff
@@ -2708,28 +2738,28 @@ static int32_t sflooper_process(CSOUND *csound, sflooper *p)
 
 static OENTRY localops[] = {
   { "sfload",S(SFLOAD),     0, 1,    "i",    "S",      (SUBR)SfLoad_S, NULL, NULL },
-  { "sfload.i",S(SFLOAD),   0, 1,    "i",    "i",      (SUBR)SfLoad, NULL, NULL },
+  { "sfload.i",S(SFLOAD),     0, 1,    "i",    "i",   (SUBR)SfLoad, NULL, NULL },
   { "sfpreset",S(SFPRESET), 0, 1,    "i",    "iiii",   (SUBR)SfPreset         },
   { "sfplay", S(SFPLAY), 0, 3, "aa", "iixxiooo",
     (SUBR)SfPlay_set, (SUBR)SfPlay     },
   { "sfplaym", S(SFPLAYMONO), 0, 3, "a", "iixxiooo",
     (SUBR)SfPlayMono_set, (SUBR)SfPlayMono },
   { "sfplist",S(SFPLIST),   0, 1,    "",     "i",      (SUBR)Sfplist          },
   { "sfilist",S(SFPLIST),   0, 1,    "",     "i",      (SUBR)Sfilist          },
-  { "sfilist.prefix",S(SFPLIST), 0, 1, "",   "iS",     (SUBR)Sfilist_prefix   },
+  { "sfilist.prefix",S(SFPLIST),   0, 1,    "",     "iS",      (SUBR)Sfilist_prefix},
 
   { "sfpassign",S(SFPASSIGN), 0, 1,  "",     "iip",    (SUBR)SfAssignAllPresets },
   { "sfinstrm", S(SFIPLAYMONO),0, 3, "a", "iixxiiooo",
     (SUBR)SfInstrPlayMono_set, (SUBR)SfInstrPlayMono },
-  { "sfinstr", S(SFIPLAY),  0, 3,    "aa", "iixxiioooo",
+  { "sfinstr", S(SFIPLAY),  0, 3,    "aa", "iixxiiooo",
```

Comment:
One last comment: seems like these OENTRY's (sfinstr, sfplay3, sfplay3m, sfinstr3, sfinstr3m) have removed the final "o" arg, which seems like it was for an undocumented iskip param. Should these be restored?

Reply:
that's right. I missed that, will do.